dev/bots/analyze.dart

```diff
@@ -173,14 +173,15 @@ Future<void> verifyDeprecations(String workingDirectory, { int minimumMatches =
           final Match match2 = _deprecationPattern2.firstMatch(lines[lineNumber]);
           if (match2 == null)
             throw 'Deprecation notice does not match required pattern.';
-          if (!lines[lineNumber].startsWith("$indent  '"))
+          if (!lines[lineNumber].startsWith(RegExp('$indent  [\'"]')))
```

Comment:
can you assert here that `indent` is safe to embed into a regexp without it being interpreted somehow? (e.g. assert that it contains nothing but spaces)